src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/StreamingSwap.tsx

```diff
@@ -1,21 +1,25 @@
 import { WarningIcon } from '@chakra-ui/icons'
 import { Progress, Stack } from '@chakra-ui/react'
+import type { TradeQuote } from '@shapeshiftoss/swapper'
 import { useTranslate } from 'react-polyglot'
 import { Row } from 'components/Row/Row'
 
 import { useThorStreamingProgress } from '../hooks/useThorStreamingProgress'
 
 export type StreamingSwapProps = {
   hopIndex: number
+  tradeId: TradeQuote['id']
 }
 
 export const StreamingSwap = (props: StreamingSwapProps) => {
-  const { hopIndex } = props
+  const { hopIndex, tradeId } = props
 
   const translate = useTranslate()
 
-  const { totalSwapCount, attemptedSwapCount, isComplete, failedSwaps } =
-    useThorStreamingProgress(hopIndex)
+  const { totalSwapCount, attemptedSwapCount, isComplete, failedSwaps } = useThorStreamingProgress(
```

Comment:
Same q. re: not passing it all the way up from `<Hops />` and select in hook

Reply:
q: Also do we want to be more explicit on the terminology for all those occurences i.e `activeQuoteId` (and `confirmedQuoteId`) vs. `tradeId` currently?

Thinking of an even safer way here (though perhaps for a follow-up) we could even make use of symbols to not only use the name as a safety belt, but also ensure that the ID we think we're referencing (activeQuote vs. confirmedQuote) *is* indeed the correct one

Reply:
@0xApotheosis keeping this one open re: naming

Reply:
big 🧠 gomes

Reply:
Naming improved in https://github.com/shapeshift/web/pull/7551/commits/5876edb802dd8822a425cfa2f970af686125116c, disambiguating `activeQuoteId` from `confirmedQuoteId`.